huxley/www/js/components/AdvisorAssignmentsView.js

```diff
@@ -230,51 +236,24 @@ var AdvisorAssignmentsView = React.createClass({
     var confirm = window.confirm("By pressing okay you are committing to the financial responsibility of each assignment. Are you sure you want to finalize assignments?");
     var school = CurrentUserStore.getCurrentUser().school;
     if (confirm) {
-      this.setState({loading: true});
-      ServerAPI.updateSchool(school.id, {assignments_finalized: true})
-        .then(this._handleFinalizedSuccess, this._handleError);
+      CurrentUserActions.updateSchool(school.id, {
+        assignments_finalized: true,
+      });
     }
   },
 
   _handleAssignmentDelete: function(assignment) {
     var confirm = window.confirm("Are you sure you want to delete this assignment?");
```

Comment:
This is for another diff, but the terminology could be better. "Decline" an assignment, perhaps?